examples/external-postgresql/values.yaml

```
@@ -0,0 +1,7 @@
+spire-server:
+  dataStore:
+    sql:
+      plugin_data:
+        database_type: "postgres"
+        connection_string: "dbname=YOURDB user=USER password=CHANGEME host=postgresql port=5432 sslmode=disable"
```

```suggestion
        host: postgresql
        port: 5432
        database: spire
        username: spire
        password: CHANGEME
        options: 
          - sslmode: disable
```

Too support the confimap combination with secrets we might want to more strongly type the values file.